worlds/messenger/docs/en_The Messenger.md

```diff
@@ -49,28 +49,28 @@ for it. The groups you can use for The Messenger are:
 ## Other changes
 
 * The player can return to the Tower of Time HQ at any point by selecting the button from the options menu
-  * This can cause issues if used at specific times. Current known:
-    * During Boss fights
-    * After Courage Note collection (Corrupted Future chase)
-      * This is currently an expected action in logic. If you do need to teleport during this chase sequence, it
-        is recommended to quit to title and reload the save
+    * This can cause issues if used at specific times. Current known:
+        * During Boss fights
+        * After Courage Note collection (Corrupted Future chase)
+            * This is currently an expected action in logic. If you do need to teleport during this chase sequence, it
+is recommended to quit to title and reload the save
 * After reaching ninja village a teleport option is added to the menu to reach it quickly
 * Toggle Windmill Shuriken button is added to option menu once the item is received
 * The mod option menu will also have a hint item button, as well as a release and collect button that are all placed when
-  the player fulfills the necessary conditions.
+the player fulfills the necessary conditions.
 * After running the game with the mod, a config file (APConfig.toml) will be generated in your game folder that can be
-  used to modify certain settings such as text size and color. This can also be used to specify a player name that can't
-  be entered in game.
+used to modify certain settings such as text size and color. This can also be used to specify a player name that can't
+be entered in game.
 
 ## Known issues
 * Ruxxtin Coffin cutscene will sometimes not play correctly, but will still reward the item
 * If you receive the Magic Firefly while in Quillshroom Marsh, The De-curse Queen cutscene will not play. You can exit
-  to Searing Crags and re-enter to get it to play correctly.
-* Sometimes upon teleporting back to HQ, Ninja will run left and enter a different portal than the one entered by the
-  player. This may also cause a softlock.
+to Searing Crags and re-enter to get it to play correctly.
+* Teleporting back to HQ, then returning to the same level you just left through a Portal can cause Ninja to run left
+and enter a different portal than the one entered by the player or other incorrect inputs, causing a soft lock
```

Comment:
A verb is probably needed here. Both "lead to" and "perform" seem fine to me, there's a few options.
```suggestion
and enter a different portal than the one entered by the player or lead to other incorrect inputs, causing a soft lock
```